qiskit/transpiler/target.py

```diff
@@ -27,6 +27,10 @@
 from qiskit.transpiler.instruction_durations import InstructionDurations
 from qiskit.transpiler.timing_constraints import TimingConstraints
 
+# import QubitProperties here to provide convenience alias for building a
+# full target
+from qiskit.providers.backend import QubitProperties  # pylint: disable=unused-import
+
```

Comment:
I don't fully understand the motivation of this line. Can you explain a bit more?

Reply:
It's a convenience thing since `QubitProperties` is now optionally part of a target I thought it would be good to have a single import for everything needed to make a taget. This is just to enable someone creating a  `Target` object to do:

```python
from qiskit.transpiler.target import Target, InstructionProperties, QubitProperties
```

instead of:

```python
from qiskit.transpiler.target import Target, InstructionProperties
from qiskit.providers.backend import QubitProperties
```
(although the latter will still work even with this.)

But, it is a bit weird to do this, so I'm not really tied to it.

Reply:
That makes a lot of sense. It's much more convenient this way.